src/styles/blocks/card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+
+  font-family: "Roboto", sans-serif;
```

Comment:
It will be more comfortable add main font-family to another file 'normalize.css' or even use variable. If you want offcourse